src/hapLrt.cpp

```diff
@@ -593,40 +596,24 @@ int main(int argc, char** argv) {
 
 
       if(populationTotal->af > 0.95 || populationTotal->af < 0.05){
-	delete populationTarget;
-	delete populationBackground;
-	delete populationTotal;
-
-	populationTarget     = NULL;
-	populationBackground = NULL;
-	populationTotal      = NULL;
+	;
+	;	
+	
 	continue;
       }
 
 
 
 	afs.push_back(populationTotal->af);
 	positions.push_back(var.position);
-	loadPhased(haplotypes, populationTotal, nsamples);
-
-	delete populationTarget;
-	delete populationBackground;
-	delete populationTotal;
+	loadPhased(haplotypes, populationTotal.get(), nsamples);
 
-	populationTarget     = NULL;
-	populationBackground = NULL;
-	populationTotal      = NULL;
+	;
```

Comment:
I find the left over three lines questionable.

Reply:
I believe that assigning these pointers to `NULL` was necessary. Previously, memory by pointer was deleted manually and then they were assigned `NULL` for some purpose, even though they continue and are not used. The destructor `unique_ptr` simply frees memory now.

Reply:
How do you think about to avoid empty statements in such source code?